static/src/stylesheets/module/atoms/_storyquestions.scss

```diff
@@ -4,6 +4,7 @@
 
     &.submeta {
         border: 0;
+        padding: ($gs-baseline/3)4;
```

Comment:
Could you please add a space between `($gs-baseline/3)` and `4` in order to make it clear this is not a string concatenation or calculation but assigns a padding in every direction?